Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
```

Comment:
The is a difference between hard and fast :) What I mean is, this project doesn't need a `xcworkspace` to run and it has no `Podfile`. So, it is easier (as in faster) to just run the thing from scratch. But if we include all the things in the `template`, that would invalidate the argument that it is faster. If you know what I mean :sunglasses: